flask_saml_sso/sso.py

```diff
@@ -74,8 +77,13 @@ def _get_saml_settings(app):
     s.setdefault('idp', {}).update(remote.get('idp'))
 
     if requests_signed:
+        if not os.path.isfile(cert_file):
+            logger.error('Unable to read cert file {}'.format(cert_file))
```

Comment:
Erm, shouldn't we handle this error case somehow? Or perhaps handle the exception instead?

Reply:
We should. I'll handle the exception.